### ISSUE TITLE
Xcode build error: does not recognize the toolset buildsystem=1

### ISSUE THREAD

**Issue by kvaDrug** [SartoxSoftware/audacium#55]
I'm trying to build audacium on macOS, Xcode 12.5.1. Getting this error:
```
CMake Error at CMakeLists.txt:134 (project):
  Generator

    Xcode

  does not recognize the toolset

    buildsystem=1

  that was specified.


-- Configuring incomplete, errors occurred!
See also "/Users/vk/Dev/3DPartySource/build/CMakeFiles/CMakeOutput.log".
```

CMakeOutput.log: 
`The system is: Darwin - 20.6.0 - x86_64`

**Comment by AnErrupTion** [comment]
I'm not familiar with macOS, is this version of Xcode recent or not? If it is then it might be the problem. Either way it's a problem in the main CMakeLists.txt file as specified by the error. I'll look at it.

**Comment by AnErrupTion** [comment]
@kvaDrug Right, so looking at https://cmake.org/cmake/help/latest/generator/Xcode.html#xcode-build-system-selection, it seems like that variable chooses the Xcode build system, and 1 is for the original build system. I would need to know what macOS version, CMake version and Xcode you have installed, and whether or not you have an M1 Mac.

**Comment by kvaDrug** [comment]
CMake 3.16.1, Intel processor.
